custom/lib_portal/wizard/library_wizard.py

```diff
@@ -1,15 +1,26 @@
-from odoo import models, fields
+from odoo import api, models, fields
 
 
 class LibraryWizard(models.TransientModel):
-    _name = 'library.wizard'
-    _description = 'Wizard for Library Module'
+    _name="lib.wizard"
```

Comment:
the name of the model is not very descriptive

Reply:
Changed it..Will go in the next commit